src/transport/CASESession.h

```diff
@@ -60,7 +85,9 @@ class DLL_EXPORT CASESession
      *
      * @return CHIP_ERROR     The result of initialization
      */
-    CHIP_ERROR WaitForSessionEstablishment(NodeId myNodeId, uint16_t myKeyId, SessionEstablishmentDelegate * delegate);
+    CHIP_ERROR WaitForSessionEstablishment(ChipCertificateSet * chipCertificateSet, const uint8_t * devOpCert,
+                                           uint16_t devOpCertLen, P256SerializedKeypair & myDeviceOpKey, NodeId myNodeId,
```

Comment:
`P256SerializedKeypair` will require the caller to provide the serialized key, even if they have the `P256Keypair` available. Maybe, just get the `P256Keypair` and let the caller deserialize, if needed. 

Same comment for `EstablishSession`.